src/utils/blockly-api.js

```diff
@@ -1,15 +1,20 @@
-// TODO: rover API
-const sendMotorCommand = (command, pin, speed) => console.log(`Motor command: ${command}, ${pin}, ${speed}`); // eslint-disable-line no-console
-const leftMotor = { FORWARD: 'XIO-P0', BACKWARD: 'XIO-P1' };
-const rightMotor = { FORWARD: 'XIO-P6', BACKWARD: 'XIO-P7' };
-const motorPins = { LEFT: leftMotor, RIGHT: rightMotor };
-
 class BlocklyApi {
-  constructor(highlightBlock, beginSleep, sensorStateCache, writeToConsole) {
+  constructor(highlightBlock, beginSleep, sensorStateCache, writeToConsole, sendToRover) {
     this.highlightBlock = highlightBlock;
     this.beginSleep = beginSleep;
     this.sensorStateCache = sensorStateCache;
     this.writeToConsole = writeToConsole;
+    this.sendToRover = sendToRover;
+  }
+
+  sendMotorCommand = (blocklyMotor, blocklyDirection, speed) => {
+    this.sendToRover(JSON.stringify({
+      type: 'motor-command',
+      'motor-id': blocklyMotor === 'LEFT' ? 'motor-left' : 'motor-right',
+      'motor-value': speed,
```

Comment:
Do these need the quotes only because of the hyphen?

Reply:
Exactly